repo/packages/B/bookkeeper/0/marathon.json.mustache

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "{{bookkeeper.app_id}}",
+  "cpus":{{bookkeeper.cpus}},
+  "mem": {{bookkeeper.mem}},
+  "instances": {{bookkeeper.instances}},
+  "constraints": [["hostname", "UNIQUE"]],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/bk/journal",
+        "hostPath": "/mnt/journal",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/index",
+        "hostPath": "/mnt/index",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/ledgers",
+        "hostPath": "/mnt/ledgers",
+        "mode": "RW"
+      },
+  
+      {
+        "containerPath": "persist",
+        "mode": "RW",
+        "persistent": {
+          "size": {{bookkeeper.psize}}
+        }
+      }
+    ],
+
+    "docker": {
+      "image": "{{resource.assets.container.docker.bookkeeper}}",
+      "network": "HOST"
+    },
+    
+    "env": {
+        "ZK": {{bookkeeper.zk_client}},
+        "BPORT": {{bookkeeper.service_port}}
```

Comment:
so you're on the host network. if you're intending to consume a **fixed** host port then you should declare that in a `portDefinitions` field. if you don't need a fixed host port, then you should still probably include a `portDefinitions` field if only to enumerate the ports that you're intending to consume. and instead of communicating the port to the docker container via `BPORT` use `PORT0` (Marathon generates indexed port variables for you)